### PR TITLE
Improved overall design of issue's section

### PR DESCRIPTION
**Pull request by abishekvashok**
As part of my task, I've done it take a look:
https://abhi2424shek.github.io/gci16.fossasia.org/

I've also added a button as stated in the task.
Please review @hemantjadon @jig08

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,6 +223,7 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
             <section class="latest-issues">
                 <h2>LATEST ISSUES</h2>
                 <div class="issues-wrapper">
+                  <div class="button"><a href=''>See all issues on&nbsp;<i class="fa fa-github"></i>github</button>
```
Comment (hemantjadon) [inline index.html]: Line is longer than allowed. (115 > 100)

*LineLengthBear, severity NORMAL, section `default`.*

Comment (hemantjadon) [inline index.html]: HTML Quotation Marks: Use only double quotes: Change the quotation mark ' to '"'.

*HTMLLintBear, severity MAJOR, section `html`.*

Comment (hemantjadon) [inline index.html]: HTML Quotation Marks: Use only double quotes: Change the quotation mark ' to '"'.

*HTMLLintBear, severity MAJOR, section `html`.*

Comment (hemantjadon) [inline index.html]: Line is longer than allowed. (115 > 100)

*LineLengthBear, severity NORMAL, section `default`.*

File: index.html [hunk 2]
```diff
@@ -157,7 +157,7 @@
                                     </a>
                                     {% endif %}
                                 </div>
-                                <p><i class="fa fa-code"></i>{{student.name}}</p>
+                                <p><i class="fa fa-code"></i>&nbsp;{{student.name}}</p>
                             </div>
```
Comment (hemantjadon) [inline index.html:161]: Protocol: Do not specify the protocol unless required: Remove the protocol "https:" from the url.

*HTMLLintBear, severity NORMAL, section `html`.*

Comment (abishekvashok) [reply index.html:161]: Fixed it

**Comment by abishekvashok** [comment]
@hemantjadon fixed everything and squashed them, now please review and merge. Task link: https://codein.withgoogle.com/dashboard/task-instances/5757239446994944/

**Comment by abishekvashok** [comment]
@hemantjadon done everything, the tests have passed now will you review @jig08 @Princu7 @daminisatya

**Comment by hemantjadon** [comment]
UI Seems much better 
Thanks For the PR 👍

**Pull request merged**